src/index.ts

```diff
@@ -19,17 +19,19 @@ const buildCtx = (opts) => {
   return ctx;
 };
 
-module.exports = (tree, opts = {}) => {
+module.exports = (tree: Object, opts: Object = {}) => {
```

Comment:
I suspect that little refactoring here can help to fix the issue with the missed exported function. Try to use this format here:
```typescript
export default (tree: object, opts: object = {}) => { ... }
```
Also, note that the `object` is lowercased, all TS types are written in lowercase.

Reply:
This workedfor building, however now I can no longer run my tests. I'm assuming this is because `exports["default"] = _default;` in the output file.

I'm also a bit confused on `Object` vs `object` vs `{}` in combination with eslint [Reference](https://mariusschulz.com/blog/the-object-type-in-typescript). Eslint does not like `: object`

Reply:
Ok, I got this working now by fixing the import sytax in the test. It seems that eslint is currently badly misconfigured, so will have to fix that next.

Reply:
When you write `Object` you refer to JS [Object class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object). And when it is written as `object` this refers to TS type of plain object with nested props.